src/components/CheckFilters.tsx

```diff
@@ -17,18 +16,14 @@ const getStyles = (theme: GrafanaTheme2) => ({
     display: flex;
     flex-direction: row;
   `,
-  marginRightSmall: css`
-    margin-right: ${theme.spacing(2)};
-  `,
   verticalSpace: css`
     margin-top: 10px;
     margin-bottom: 10px;
-    margin-right: ${theme.spacing(2)};
   `,
 });
 
 interface Props {
-  handleResetFilters: () => void;
+  onReset: () => void;
```

Comment:
Also thought I'd try and standardise so that props have `on${Whatever}` and `handle${Whatever}` functions are defined within a component that need to add extra steps to `on${Whatever}`.